changelog/unreleased/issue-2203

```diff
@@ -0,0 +1,11 @@
+Bugfix: Fix reading passwords from stdin
+
+Passwords for `init` and `key add` can now be read from non-terminal stdin
+without having to pad the input to 1000 characters.
```

Comment:
Please leave out the second half of the sentence, we're trying to write the changelog for users, and that's a (rather obscure) implementation detail :)